cli/command_snapshot_create.go

```diff
@@ -69,6 +70,7 @@ func (c *commandSnapshotCreate) setup(svc appServices, parent commandParent) {
 	cmd.Flag("tags", "Tags applied on the snapshot. Must be provided in the <key>:<value> format.").StringsVar(&c.snapshotCreateTags)
 	cmd.Flag("pin", "Create a pinned snapshot that's will not expire automatically").StringsVar(&c.pins)
 	cmd.Flag("flush-per-source", "Flush writes at the end of each source").Hidden().BoolVar(&c.flushPerSource)
+	cmd.Flag("set-source", "Override the source of the snapshot.").StringVar(&c.sourceOverride)
```

Comment:
Perhaps `override-source` ?

Reply:
This may be confusing since _"source"_ has a specific meaning in kopia, a snapshot source is the triplet `{host, user, path}`

Given that the intend here is to override the path, perhaps this should be `override-source-path` or `override-path`.

Reply:
Right now it allows overriding all three parts of the triplet (using the same logic to determine what you mean as list-snapshots, etc.)
https://github.com/kopia/kopia/pull/3041/files#diff-30637363ab70c6f1f34641e0e058998b680960f18a89107285161e3d479f04b6R745

Reply:
Got it. When I skimmed over the PR, I incorrectly assumed it was only overriding the `path`.